content/news/2023/10/2023-10-24-navigating-the-nist-industrial-control-systems-overlay.md

```diff
@@ -53,3 +53,43 @@ NIST guidance can often be daunting to digest; the ICS guidebook is nearly 250 p
 ## Disclaimer
 
 *The EPA is not being represented by the author of this publication. The United States government and the EPA do not endorse any commercial or open-source products or services. Any reference to specific products, processes, or services by service mark, trademark, manufacturer, or otherwise is provided for informational purposes and does not constitute or imply their endorsement, recommendation, or favoring by the United States government and the EPA.*
+
+{{< note variant="Activity" >}}
+Activity text goes here...
+{{< /note >}}
+
+{{< note variant="Action" >}}
+Action text goes here...
+{{< /note >}}
+
+{{< note variant="Alert" >}}
+Alert text goes here...
+{{< /note >}}
+
+{{< note variant="Comment" >}}
+Comment text goes here...
+{{< /note >}}
+
+{{< note variant="Video" >}}
+Video text goes here...
+{{< /note >}}
+
+{{< note variant="Who can Join?" >}}
+Group text goes here...
+{{< /note >}}
+
+{{< note variant="Note" >}}
+Note text goes here...
+{{< /note >}}
+
+{{< note variant="Disclaimer" >}}{{< /note >}}
```

Comment:
1. Let's keep the previous convention and use lowercase for the argument:

```
{{< note variant="comment" >}}
Comment text goes here...
{{< /note >}}
```

2. Use `join` to make it easier for content editors:

```
{{< note variant="join" >}}
Group text goes here...
{{< /note >}}
```

Reply:
udpates made for lower case both in this file and in the shortcode file

Reply:
actually after fixing i think we need to keep it captlize casue when we change it to lower case the component will render that way...